### PR TITLE
New terminal UI

### PR DESCRIPTION
**Pull request by vivianedias**

### CODE REVIEW

**Comment by sembrestels** [comment]
It's amazing work @vivianedias, could you take care of these four things before we merge?

- [ ] Routes should change when clicking a link that moves from the terminal to the front page and vice-versa 
- [ ] Buttons should go through hover and clicked states
- [ ] Please remove the two green blocks in library (in the figma they were parts of the underlying buttons)
- [ ] Library button moves slightly when it's hovered, it shouldn't

**Comment by vivianedias** [comment]
> It's amazing work @vivianedias, could you take care of these four things before we merge?
> 
>     * [ ]  Routes should change when clicking a link that moves from the terminal to the front page and vice-versa
> 
>     * [x]  Buttons should go through hover and clicked states
> 
>     * [x]  Please remove the two green blocks in library (in the figma they were parts of the underlying buttons)
> 
>     * [x]  Library button moves slightly when it's hovered, it shouldn't

All the requested changes are done! I even updated the switch colors to the newest ones. That first question tho about the routes, they seem to be working fine here. Did you update the libs? I also updated react-router in the other PR. Thanks for the feedback!

File: packages/evmcrispr-terminal/src/utils/local-storage.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import type { type Script } from '../components/library-scripts';
```
Comment (kamikazebr) [inline packages/evmcrispr-terminal/src/utils/local-storage.ts:1]: The Vercel error

image

Comment (vivianedias) [reply packages/evmcrispr-terminal/src/utils/local-storage.ts]: thanks for the help! @kamikazebr I think its working now :)

**Pull request merged**